build.gradle

```diff
@@ -18,10 +18,11 @@ repositories {
         name = "sonatype"
```

Comment:
UPDATE THE VERSION OF THE PLUGIN TO `0.7.0`

Reply:
Did it